src/xpk/core/core.py

```diff
@@ -2104,8 +2104,12 @@ def get_main_container(args, system, docker_image, resource_type) -> str:
         ' is required but not installed. Aborting"; exit 24;};'
     )
     xpk_internal_commands += (
+        'set -x;'
         'WORKER_ID=$HOSTNAME;'
-        f'gsutil -m cp -r /tmp/xla_dump/ {args.debug_dump_gcs}/$WORKER_ID;'
+        f'gcloud storage cp -r /tmp/xla_dump/ {args.debug_dump_gcs}/$WORKER_ID;'
+        f'gcloud storage cp /tmp/*.dump {args.debug_dump_gcs}/$WORKER_ID;'
```

Comment:
Do we need additional flags set to find the /tmp/*.dump and /tmp/tpu_logs? just want to make sure these additional dumps always exist and don't crash a users job if they can't be found (like only available with additional flags)

Reply:
1. No flags for dumping /tmp/tpu_logs.   
2. As for `/tmp/*.dump`, this file will be dumped in case of some specific hardware failure. We can also force core dump with some flags as well.

There will be an error without finding the specific path. And the pipeline will still continue after the error since we didn't `set -e;`.

Reply:
> We can also force core dump with some flags as well.

SG, Should we explicitly set the flags for the core dumps?